gst/nnstreamer/meson.build

```diff
@@ -99,3 +127,19 @@ nnstreamer_dep = declare_dependency(link_with: nnstreamer_lib,
 install_headers(nnstreamer_headers,
   subdir: 'nnstreamer'
 )
+
+if not (meson.project_name() == 'nnstreamer')
```

Comment:
How about moving configuration into this file? (not the meson script in root dir)